torch/csrc/jit/codegen/cuda/compute_at.cpp

```diff
@@ -176,7 +176,7 @@ void ComputeAt::runAt(
       " are not in the same fusion.");
 
   if (mode == ComputeAtMode::MostInlined) {
-    consumer_position = consumer->nDims();
+    consumer_position = -1;
```

Comment:
There is no real change here. I am just abusing `ComputeAt` here as a unit test.